src/cameras/OrthographicCamera.js

```diff
@@ -134,7 +130,8 @@ OrthographicCamera.prototype = Object.assign( Object.create( Camera.prototype ),
 
 	}
 
-} );
+}
 
+OrthographicCamera.prototype.isOrthographicCamera = true;
```

Comment:
could you use the `Object.defineProperty()` pattern demonstrated in #20110? 🙏

Reply:
> could you use the `Object.defineProperty()` pattern demonstrated in #20110? 🙏

OK.